index.html

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>E-com whitepaper</title>
+    <link rel="stylesheet" href="../Sass/style.css" />
+    <link
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.3.0/css/font-awesome.css"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="../Sass/style.css" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
+      crossorigin="anonymous"
+    />
+  </head>
+  <body>
+    <header>
+      <nav class="mobileNav d-md-none">
+        <i class="fa fa-bars pt-4 fa-3x" id="hamburger" aria-hidden="true"></i>
+      </nav>
+      <nav class="desktopNav mt-3">
+        <div
+          class="destopNavItems d-flex align-items-center justify-content-between"
+        >
+          <div class="logo ms-5 d-none d-md-block">
+            <a class="logo" href="./index.html"
+              ><img src="../img/logo.jpeg" alt=""
+            /></a>
+            <br />
+          </div>
+          <div id="desktopList" class="d-none d-md-flex">
+            <img
+              id="closeX"
+              class="d-none ps-5 mb-5"
+              src="../img/close.png"
+              alt=""
+            />
```

Comment:
- [ ] Just like the other reviewer highlighted, you forgot to include the top navbar for this project. The top navbar contains social media icons and some other links and has a dark ash background (`#272a31`). The image below will help you to understand the feature you didn't implement:

![image](https://user-images.githubusercontent.com/77106962/181357002-833c84e5-ae4c-4223-acee-cf183d4da8ac.png)

Reply:
top nav bar added